applications/adv_networking_bench/adv_networking_bench_default_tx_rx_multi_q_hds.yaml

```diff
@@ -0,0 +1,334 @@
+          - Headers_RX_CPU
+          - CH4_VRT_Headers_RX_CPU
+          - CH4_Data_RX_GPU
+        - name: rx_q_5
+          id: 5
+          cpu_core: 9
+          batch_size: 256
+          memory_regions:
+          - Headers_RX_CPU
+          - CH5_VRT_Headers_RX_CPU
+          - CH5_Data_RX_GPU
+        - name: rx_q_6
+          id: 6
+          cpu_core: 9
+          batch_size: 256
+          memory_regions:
+          - Headers_RX_CPU
+          - CH6_VRT_Headers_RX_CPU
+          - CH6_Data_RX_GPU
+        - name: rx_q_7
+          id: 7
+          cpu_core: 9
+          batch_size: 256
+          memory_regions:
+          - Headers_RX_CPU
+          - CH7_VRT_Headers_RX_CPU
+          - CH7_Data_RX_GPU
+        - name: rx_q_8
+          id: 8
+          cpu_core: 9
+          batch_size: 256
+          memory_regions:
+          - Headers_RX_CPU
+          - CH8_VRT_Headers_RX_CPU
+          - CH8_Data_RX_GPU
+        flows:
+        - name: flow_4097
+          id: 0
+          action:
+            type: queue
+            id: 0
+          match:
+            udp_src: 4097
+            udp_dst: 4097
+        - name: flow_4096
+          id: 1
+          action:
+            type: queue
+            id: 1
+          match:
+            udp_src: 4096
+            udp_dst: 4096
+        - name: flow_4095
+          id: 2
+          action:
+            type: queue
+            id: 2
+          match:
+            udp_src: 4095
+            udp_dst: 4095
+        - name: flow_4094
+          id: 3
+          action:
+            type: queue
+            id: 3
+          match:
+            udp_src: 4094
+            udp_dst: 4094
+        - name: flow_4093
+          id: 4
+          action:
+            type: queue
+            id: 4
+          match:
+            udp_src: 4093
+            udp_dst: 4093
+        - name: flow_4092
+          id: 5
+          action:
+            type: queue
+            id: 5
+          match:
+            udp_src: 4092
+            udp_dst: 4092
+        - name: flow_4091
+          id: 6
+          action:
+            type: queue
+            id: 6
+          match:
+            udp_src: 4091
+            udp_dst: 4091
+        - name: flow_4090
+          id: 7
+          action:
+            type: queue
+            id: 7
+          match:
+            udp_src: 4090
+            udp_dst: 4090
+        - name: flow_4089
+          id: 8
+          action:
+            type: queue
+            id: 8
+          match:
+            udp_src: 4089
+            udp_dst: 4089
+bench_rx:
+  interface_name: rx_port
+  gpu_direct: true
+  split_boundary: true
+  batch_size: 1
+  max_packet_size: 1064
+  header_size: 170
+  reorder_kernel: false
+bench_tx:
+  interface_name: tx_port
+  gpu_direct: false
+  split_boundary: 0
+  batch_size: 10240
+  payload_size: 1000
+  header_size: 64
+  eth_dst_addr: 48:b0:2d:f4:04:48
+  ip_src_addr: <1.2.3.4>
+  ip_dst_addr: <5.6.7.8>
+  udp_src_port: 4089-4097
```

Comment:
Should we authorize a native yaml list?

Reply:
This is really only for a test app so I don't know if we need to make this more complicated.